.github/workflows/compliance.yml

```diff
@@ -0,0 +1,18 @@
+name: Compliance
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  compliance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mtfoley/pr-compliance-action@11b664f0fcf2c4ce954f05ccfcaab6e52b529f86
```

Comment:
I think it'd be good to disable `issue-labels-auto-close`, and add the same exceptions that you added in neostandard for `renovate`

Reply:
👍 This report uses dependabot currently though and I can't remember why/when the ignores are handy 😅

Reply:
@scagood `issue-labels-auto-close` will not have to be disabled as no `issue-labels` has been set :)